src/pages/Main/MainCarousel.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import ImgCom from './ImgCom';
+
+class MainCarousel extends Component {
+  constructor() {
+    super();
+    this.state = {
+      firstImg: 0,
+      sliderArr: [],
+      info: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/visualData.json')
```

Comment:
- 주소에서 localhost 부분은 생략해 주시는 게 더 좋습니다. 한번에 여러개의 서버를 띄워둘 경우, 포트 번호가 변경이 되게 되는데, 지금 처럼 3000 포트로 명시를 해두면 fetch 함수들을 일일이 수정하셔야 합니다. 생략을 하게되면, 그때그때의 포트 번호가 자동으로 들어가게 되서 더 편합니다.
```suggestion
    fetch('/data/visualData.json')
```